multiqc/modules/fastqc/fastqc.py

```diff
@@ -556,7 +557,52 @@ def seq_dup_levels_plot (self):
                         plots.linegraph.plot(data, pconfig)
         })
         
-        
+    
+    def overrepresented_sequences (self):
+        """Sum the percentages of overrepresented sequences and display them in a bar plot"""
+
+        data = dict()
+        for s_name in self.fastqc_data:
+            data[s_name]=dict()
+            try:
+                total_pcnt = sum( [ float(d['percentage']) for d in self.fastqc_data[s_name]['overrepresented_sequences']] )
+                data[s_name]['overrepresented']=total_pcnt
+            except KeyError:
+                continue 
```

Comment:
Better to use `log.warning` or `log.debug` with an explanation of what's gone wrong and for which sample..